src/game/mario_actions_airborne.c

```diff
@@ -2007,7 +2007,7 @@ s32 mario_execute_airborne_action(struct MarioState *m) {
         return TRUE;
     }
 
-#ifndef NO_FALL_DAMAGE_SOUND
+#ifndef NO_FALL_DAMAGE
```

Comment:
Personally not a fan, just remove NO_FALL_DAMAGE_SOUND's effect entirely (this is not hard to change yourself)

Reply:
agreed tbh